fusion-cli/get-test-files.js

```diff
@@ -28,12 +28,19 @@ function getFiles() /*: Array<string> */ {
       '**/*.test.js',
       ...testPathIgnorePatterns.map(pattern => `!${pattern}`),
     ],
-    {gitignore: true}
+    {
+      gitignore: true,
+      ignore: ['node_modules']
+    }
   );
 
-  const filesPerJob = Math.ceil(allFiles.length / JOB_COUNT);
+  const filesPerJob = Math.floor(allFiles.length / JOB_COUNT);
```

Comment:
If Math.ceil was rounding more than the number of tests per job, the last job was not being passed any paths. This was causing the final job to run all fusion-cli tests at once and timing out